databricks/koalas/tests/test_dataframe.py

```diff
@@ -2375,8 +2375,8 @@ def test_drop_duplicates(self):
 
     def test_reindex(self):
         index = ["A", "B", "C", "D", "E"]
-        pdf = pd.DataFrame({"numbers": [1.0, 2.0, 3.0, 4.0, 5.0]}, index=index)
-        kdf = ks.DataFrame({"numbers": [1.0, 2.0, 3.0, 4.0, 5.0]}, index=index)
+        pdf = pd.DataFrame({"numbers": [1.0, 2.0, 3.0, 4.0, None]}, index=index)
+        kdf = ks.DataFrame({"numbers": [1.0, 2.0, 3.0, 4.0, None]}, index=index)
```

Comment:
nit: Would you mind use `ks.from_pandas(pdf)` so that we can reuse the existing `pdf` internally ?

Reply:
Yes, done